### PR TITLE
Скрипаль А.Д. 381806-2. ЛР3

### PR DESCRIPTION
**Pull request by drwff**
### Pull Request Readiness Checklist

See details at [code review]( https://github.com/UNN-ITMM-Software/devtools-course-practice/blob/main/lab-guide/lab-1-setup.md#%D1%80%D0%B5%D1%86%D0%B5%D0%BD%D0%B7%D0%B8%D1%8F-%D0%BA%D0%BE%D0%B4%D0%B0)

- [ ] Code is well documented and more information can be seen on the [website](https://devtools-course-practice-2021.readthedocs.io/en/latest/modules.html) (recommended)
- [ ] Make sure that the testing has passed successfully on [GitHub Actions](https://github.com/UNN-ITMM-Software/devtools-course-practice/actions)
- [ ] Get two approve from your fellow students
- [ ] Take a review from one of the administrators

### CODE REVIEW

**Comment by drwff** [comment]
@mercy131 @Sergo567 проверьте

File: modules/intersectLineAndPlane/src/liap_app.cpp [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+#include "include/ilap.h"
+
+PointApp::PointApp() : message_("") {}
+
+void PointApp::help(const char* appname, const char* message) {
+    message_ =
+        std::string(message) +
+        "This is a intersection line and plane application.\n\n" +
+        "Please provide arguments in the following format:\n$"
+        + appname + "x1, y1, z1, ... x5, y5, z5 \n\n"  \
```
Comment (Sergo567) [inline modules/intersectLineAndPlane/src/liap_app.cpp:21]: Нужно более подробно описать за что отвечают координаты.

File: modules/intersectLineAndPlane/src/liap_app.cpp [hunk 2]
```diff
@@ -0,0 +1,103 @@
+// Copyright 2021 Skripal Andrey
+#include <stdlib.h>
+
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <utility>
+#include <cmath>
```
Comment (mercy131) [inline modules/intersectLineAndPlane/src/liap_app.cpp:8]: Лучше убрать лишние библиотеки.

**Review by mercy131 — APPROVED** [review]

**Review by Sergo567 — APPROVED** [review]

**Review by OnlyDeniko — APPROVED** [review]
👍

**Pull request merged**